google_shopping_functions.js

```diff
@@ -21,9 +77,20 @@ var data = require("./products.json")
 //
 
 
+getItems(data);
+
+
+/*Part 5 - All items made by Sony.*/
+console.log(getItemsByBrand(getItems(data), 'sony'));
```

Comment:
Instead of calling getItems(data) each time, you can store it in a variable like this:
```
var items = getItems(data);
...
console.log(getItemsByBrand(items, 'sony');
```